core/domain/wipeout_service.py

```diff
@@ -1010,6 +1034,67 @@ def _remove_user_id_from_models_transactional(summary_models):
                 i:i + feconf.MAX_NUMBER_OF_OPS_IN_TRANSACTION])
 
 
+def _pseudonymize_app_feedback_report_models(pending_deletion_request):
+    """Pseudonymize the app feedback report models for the user with user_id,
+    if they scrubbed a feedback report. If the user scrubs multiple reports,
+    they will be given the same pseudonym for each model entity.
+
+    Args:
+        pending_deletion_request: PendingDeletionRequest. The pending deletion
+            request object to be saved in the datastore.
+    """
+    model_class = app_feedback_report_models.AppFeedbackReportModel
+    user_id = pending_deletion_request.user_id
+
+    feedback_report_models = model_class.query(
+        model_class.scrubbed_by == user_id).fetch()
+    report_ids = set([model.id for model in feedback_report_models])
+
+    # Fill in any missing keys in the category's
+    # pseudonymizable_entity_mappings; this is the same as
+    # _save_pseudonymizable_entity_mappings(), but using the same
+    # pseudonym for each entity so that a user will have the same pseudonymized
+    # ID for each entity referencing them.
```

Comment:
Yeah, I think ideally this would be mapping each user ID to the PID but that introduces some indirection in that there is not mapping from the actual entity to the PID it uses.